crates/query-engine/metadata/src/metadata/mutations.rs

```diff
@@ -8,4 +8,5 @@ use serde::{Deserialize, Serialize};
 #[serde(rename_all = "camelCase")]
 pub enum MutationsVersion {
     V1,
+    VeryExperimentalWip,
```

Comment:
It's excellent that we can iterate on the design in this way.

Maybe it would have been better to pick a different name for these new experimental versions though. Consider which name we should pick the next time we want to try a different experiment. `VeryExperimentalWip` is a bit reminiscient of document file name versions like `report_final.doc` and report_final_2.doc` if you know the meme :-).

Reply:
In future, perhaps something like `unsafe_next`? We only ever want one of these versions at once, and it communicates that if you use it, all stability bets are off?